### PR TITLE
Added eslint and prettier, fixed grid, fixed layout by moving element…

### PR DESCRIPTION
**Pull request by wojtasiq**
…s to separate components, linting whole client

### CODE REVIEW

File: .github/workflows/lint_client.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      with:
+        node-version: '13.12'
+    - name: Install dependency
+      run: |
+        cd client
+        yarn install
+    - name: Lint
+      run: |
+        cd client
+        yarn lint
```
Comment (baatochan) [inline .github/workflows/lint_client.yml:21]: rly? could we keep EOF newline enforcement?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.vscode
+.idea
```
Comment (baatochan) [inline .gitignore:2]: same as above

File: .github/workflows/lint_client.yml (hunk 1 above)
Comment (wojtasiq) [reply .github/workflows/lint_client.yml:21]: ok, i can add new lines, but there is no way to enforce that in repository configuration/.gitignore files.
In future i will  try to remember to add eof new lines ;)

File: client/.editorconfig [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[*.{js,jsx,ts,tsx,vue}]
+indent_style = space
+indent_size = 2
```
[the remark this answers is not in the document]
Comment (wojtasiq) [reply client/.editorconfig:3]: file auto-created by cli-plugin-eslint. I chosen default preset ;P
If don't mind changing to tab's ;)

File: .gitignore (hunk 2 above)
Comment (wojtasiq) [reply .gitignore]: added new line

File: client/.editorconfig (hunk 3 above)
[the remark this answers is not in the document]
Comment (baatochan) [reply client/.editorconfig:3]: You see - I use tabs, I'm okay with spaces tho, but why 2 spaces and not default 4 XD

[the remark this answers is not in the document]
Comment (wojtasiq) [reply client/.editorconfig:3]: oh, i know why 2 spaces - 4 spaces in component could make you need to scroll horizontally
We propose we should stay on 2 spaces. You can write tabs and then run `yarn lint-fix` ;)

**Comment by wojtasiq** [comment]
@Isild please don't merge when there are conversation opened xD

**Pull request merged**